internal/config/config.go

```diff
@@ -19,28 +19,33 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
-type Tokens struct {
-	GithubToken string `yaml:"githubToken"`
-	JiraToken   string `yaml:"jiraToken"`
+type Config struct {
+	JiraBaseURL string `yaml:"jiraBaseURL"`
+	AuthTokens  struct {
+		JiraToken   string `yaml:"jira"`
+		GithubToken string `yaml:"github"`
+	} `yaml:"authTokens"`
 }
 
-func ReadTokensYaml(file string) (*Tokens, error) {
+func ReadConfigYaml(file string) (*Config, error) {
```

Comment:
nit: yaml is the format and not relevant to the method, so I'd prefer to just call is `ReadConfig` or even `ReadFile`.